todoman/ui.py

```diff
@@ -100,6 +99,18 @@ def __init__(self, todo, lists, formatter):
                         in widgets.ExtendedEdit.HELP)
         )
 
+        def change_current_list(radio_button, new_state, new_list):
+            self.current_list = new_list
+
+        list_selector = []
+        for _list in self.lists:
+            urwid.RadioButton(list_selector,
+                              _list.name == self.current_list.name,
+                              state=_list is self.current_list,
```

Comment:
I rather meant that you should write `RadioButton(list_selector, _list.name, state=_list.name == self.current_list.name, ...)`